src/constants/ActionTypes.js

```diff
@@ -12,16 +12,17 @@ export const EDITOR_REFRESH = "EDITOR_REFRESH";
 export const EDITOR_RECOVER = "EDITOR_RECOVER";
 export const EDITOR_CHANGE = "EDITOR_CHANGE";
 export const EDITOR_UPDATE_SAVEDTEXT = "EDITOR_UPDATE_SAVEDTEXT";
-export const ADD_PW = "ADD_PW";
 export const SAVE_START = "SAVE_START";
+//unused constant
```

Comment:
What's the unused constant?

Reply:
Those were the constants that weren't used it anywhere else. I thought there were some intention behind it so I didn't delete it but it's probably best to remove it.